EIPS/eip-4762.md

```diff
@@ -132,11 +132,6 @@ When a contract creation is initialized, process these write events:
 
 ```
 (contract_address, 0, BASIC_DATA_LEAF_KEY)
-```
```

Comment:
I wonder if "creation is initialized" is well defined. As in: after collision check.

Reply:
I was understanding it as in when the code is set against the contract address after being initialized. That's when hash is computed at least for us